themes/src/main/resources-community/theme/base/email/messages/messages_nl.properties

```diff
@@ -1,27 +1,27 @@
-emailVerificationSubject=Bevestig emailadres
-emailVerificationBody=Iemand heeft een {2} account aangemaakt met dit emailadres. Indien u dit was, klikt u op de onderstaande koppeling om uw emailadres te bevestigen \n\n{0}\n\nDeze koppeling zal binnen {1} minuten vervallen.\n\nU kunt dit bericht negeren indien u dit account niet heeft aangemaakt.
-emailVerificationBodyHtml=<p>Iemand heeft een {2} account aangemaakt met dit emailadres. Indien u dit was, klikt u op de onderstaande koppeling om uw emailadres te bevestigen</p><p><a href="{0}">Koppeling naar email-adres bevestiging</a></p><p>Deze koppeling zal binnen {1} minuten vervallen.</p<p>U kunt dit bericht negeren indien u dit account niet heeft aangemaakt.</p>
+emailVerificationSubject=Bevestig e-mailadres
+emailVerificationBody=Iemand heeft een {2} account aangemaakt met dit e-mailadres. Als u dit was, klikt u op de onderstaande koppeling om uw e-mailadres te bevestigen \n\n{0}\n\nDeze koppeling zal binnen {1} minuten vervallen.\n\nU kunt dit bericht negeren indien u dit account niet heeft aangemaakt.
+emailVerificationBodyHtml=<p>Iemand heeft een {2} account aangemaakt met dit e-mailadres. Als u dit was, klikt u op de onderstaande koppeling om uw e-mailadres te bevestigen</p><p><a href="{0}">Koppeling naar e-mailadres bevestiging</a></p><p>Deze koppeling zal binnen {1} minuten vervallen.</p<p>U kunt dit bericht negeren indien u dit account niet heeft aangemaakt.</p>
 emailTestSubject=[KEYCLOAK] - SMTP testbericht
 emailTestBody=Dit is een testbericht
 emailTestBodyHtml=<p>Dit is een testbericht</p>
 identityProviderLinkSubject=Koppel {0}
-identityProviderLinkBody=Iemand wil uw "{1}" account koppelen met "{0}" account van gebruiker {2}. Indien u dit was, klik dan op de onderstaande link om de accounts te koppelen\n\n{3}\n\nDeze link zal over {4} minuten vervallen.\n\nIndien u de accounts niet wilt koppelen, negeert u dan dit bericht. Als u accounts koppelt, dan kunt in naar {1} inloggen via {0}.
-identityProviderLinkBodyHtml=<p>Iemand wil uw "{1}" account koppelen met "{0}" account van gebruiker {2}. Indien u dit was, klik dan op de onderstaande link om de accounts te koppelen</p><p><a href="{3}">Link om accounts te koppelen</a></p><p>Deze link zal over {4} minuten vervallen.</p><p>Indien u de accounts niet wilt koppelen, negeert u dan dit bericht. Als u accounts koppelt, dan kunt in naar {1} inloggen via {0}.</p>
+identityProviderLinkBody=Iemand wil uw "{1}" account koppelen met "{0}" account van gebruiker {2}. Als u dit was, klik dan op de onderstaande link om de accounts te koppelen\n\n{3}\n\nDeze link zal over {4} minuten vervallen.\n\nAls u de accounts niet wilt koppelen, negeer dan dit bericht. Als u accounts koppelt, dan kunt u bij {1} inloggen via {0}.
```

Comment:
true, but better to change this consistently with the other translations as well.